_layouts/book.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: wide
+---
+
+{% include components/book-presenter.html book=page.book %}
+
+<section>
+  <div class="full-width-rule">
+    <div class="hex">
+      <span>What you will learn</span>
+    </div>
+  </div>
+  {% include components/image-list.html items=page.what_you_will_learn %}
+  <div class="full-width-rule">
+  </div>
+</section>
```

Comment:
**issue:** I think this section needs more horizontal spacing between its elements (compare invision design). `--block-flow: var(--block-flow-md)` would do, but that also affects the sections own top margin.
We could introduce a custom class for a section with bigger block spacing.
However, I think we're just missing a general rule for `.full-width-rule`. The blockflow mixing already has some additional spacing around `hr` and the same should probably apply to `.full-width-rule`.
So the diff for that would be:
```diff
--- i/_sass/mixins/_blockflow.scss
+++ w/_sass/mixins/_blockflow.scss
@@ -5,13 +5,13 @@
     + * {
       margin-block-start: var(--block-flow, $width);

-      &:is(hr) {
+      &:is(hr, .full-width-rule) {
         margin-block-start: var(--block-flow-md);
       }
     }
   }

-  :where(& > hr + *) {
+  :where(& > :is(hr, .full-width-rule) + *) {
     margin-block-start: var(--block-flow-md);
   }
```